packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -63,5 +63,6 @@
   "force_source_retrieve_display_text": "Retrieve Source from Org",
   "force_diff_against_org": "SFDX: Diff File Against Org",
   "force_analytics_template_create_text": "SFDX: Create Sample Analytics Template",
-  "experimental_deploy_retrieve_description": "Enable performance enhancements to the deploy/retrieve experience."
-}
\ No newline at end of file
+  "experimental_deploy_retrieve_description": "Enable performance enhancements to the deploy/retrieve experience.",
+  "experimental_cli_command_description": "Revert to CLI implementation of Apex commands"
```

Comment:
@ekapner For this PR we're adding a new flag to control switching between the CLI implementation for Apex commands and the new library that we're replacing that with. Right now we've named the flag as `cliCommand` under the `experimental` category and the description is above ^^. Please let me know if you have any suggestions for the name or description for the flag, thanks!

Reply:
Here's what it would look like for the user 
<img width="490" alt="Screen Shot 2020-07-15 at 11 09 08 AM" src="https://user-images.githubusercontent.com/21326913/87568843-e8fefc00-c68b-11ea-85e9-a71a7ce33e41.png">

Reply:
@AnanyaJha - pinged you in the work item - need some background info before I can provide feedback.

Reply:
How about Experimental: Use Apex Library